src/__tests__/MentorDashboard.test.js

```diff
@@ -1,240 +1,3 @@
-// import React from 'react';
-// import { render, cleanup, screen, waitFor } from '@testing-library/react';
-// import userEvent from '@testing-library/user-event';
-// import { act } from 'react-dom/test-utils';
-// // import HomeContainer from '../components/pages/Home/HomeContainer';
-// import createTestStore from '../__mocks__/CreateTestStore';
-// import { Provider } from 'react-redux';
-// import SkeletonLoadingComponent from '../components/common/SkeletonLoading';
-// import Sidebar from '../components/common/Sidebar/Sidebar.js';
-
-// afterEach(cleanup);
-// // creating store variable
-// let store;
-// // creating a mock useOktaAuth, needed this so we can log in
-// jest.mock('@okta/okta-react', () => ({
-//   useOktaAuth: () => {
-//     return {
-//       authState: {
-//         isAuthenticated: true,
-//       },
-//       authService: {
-//         getUser: jest.fn(() => {
-//           return Promise.resolve({
-//             sub: '00ultwqjtqt4VCcS24x6',
-//             name: 'Test004 User',
-//             email: 'llama004@maildrop.cc',
-//             preferred_username: 'llama004@maildrop.cc',
-//             role: 3,
-//           });
-//         }),
-//       },
-//     };
-//   },
-// }));
-// // creating a mock action
-// jest.mock('../state/actions/index', () => ({
-//   getUserProfile: jest.fn(() => {
-//     return {
-//       type: 'USER_PROFILE',
-//       payload: {
-//         sub: '00ultwqjtqt4VCcS24x6',
-//         name: 'Test004 User',
-//         email: 'llama004@maildrop.cc',
-//         preferred_username: 'llama004@maildrop.cc',
-//         role: 3,
-//       },
-//     };
-//   }),
-// }));
-
 test('Home Container no longer exists, this entire test needs to be revisited/refactored or deleted. It was originally for the Mentor Dashboard Test', () => {
```

Comment:
add .skip to this line like so `test.skip()`. leaving in the console log is fine for.